set.mm

```diff
@@ -114701,10 +114701,25 @@ it could represent the (meaningless) operation of
      10-Feb-1995.) $)
   df-neg $a |- -u A = ( 0 - A ) $.
 
-  $( Alternate proof of ~ 0cn which does not reference ~ ax-1cn .  (Contributed
-     by NM, 19-Feb-2005.)  (Revised by Mario Carneiro, 27-May-2016.)
+  ${
+    $d x y $.  $d x z $.
+    $( Alternate proof of ~ 0cn which does not reference ~ ax-1cn .
+       (Contributed by NM, 19-Feb-2005.)  (Revised by Mario Carneiro,
+       27-May-2016.)  Reduce dependencies on axioms.  (Revised by Steven
+       Nguyen, 7-Jan-2022.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    0cnALT $p |- 0 e. CC $=
+      ( vx vy vz ci cc wcel cv cmul co caddc wceq cr wrex ax-icn cnre ax-rnegex
+      cc0 wa readdcl eleq1 syl5ibcom rexlimdva mpd adantr rexlimiva mp2b recni
+      ) QDEFDAGZDBGHIJIKBLMZALMQLFZNABDOUIUJALUHLFZUJUIUKUHCGZJIZQKZCLMUJCUHPUK
+      UNUJCLUKULLFRUMLFUNUJUHULSUMQLTUAUBUCUDUEUFUG $.
+  $}
+
+  $( Obsolete version of ~ 0cnALT as of 7-Jan-2022.  Alternate proof of ~ 0cn
+     which does not reference ~ ax-1cn .  (Contributed by NM, 19-Feb-2005.)
+     (Revised by Mario Carneiro, 27-May-2016.)
      (Proof modification is discouraged.)  (New usage is discouraged.) $)
-  0cnALT $p |- 0 e. CC $=
+  0cnALTOLD $p |- 0 e. CC $=
     ( vx ci cv caddc co cc0 wceq wrex wcel ax-icn cnegex ax-mp addcl mpan eleq1
     cc syl5ibcom rexlimiv ) BACZDEZFGZAPHZFPIZBPIZUBJABKLUAUCAPSPIZTPIZUAUCUDUE
     UFJBSMNTFPOQRL $.
```

Comment:
This may reduce axiom dependencies (which ones?) but this lengthens the proof. Keep as 0cnALT2 ?

Reply:
it saves {8 13 sep nul pow pr un 1cn addcl mulcl mulrcl mulcom addass mulass distr i2m1 1ne0 1rid rrecex pre-lttri pre-lttrn pre-ltadd}

sure, although [currently 0cnALT] uses ax-1cn somehow so the comment isn't accurate anymore, how do I describe the new 0cnALT2?

Reply:
OK now